### PR TITLE
Markor v2.9 update release

### PR DESCRIPTION
**Pull request by gsantner**
This pull request is about release of the next major Markor version, planned for release in early may. This time also with Changelog, blog post etc. This pull request is the preparation.  

Recently tried to mention you in merge commits when you contributed to the project - so when reaching here - can copy the git history and you are automatically there standing by your contribution. So thanks for helping!

**Now I also invite you to show what you did, worked on, fixed, added as new feature. If you want of course.**

* I added the last Markor release blog post (written in Markdown as well!) to the repository as reference.  --> <https://github.com/gsantner/markor/tree/release/v2.9.0/doc/news>.  Also copied the 2.8 release write-up to the new 2.9 write-up. 
* Also added a section to the CHANGELOG - this is short and text only (no image) - summary for app store changelog like f-droid.

To suggest something to be added to the post or changelog, you can easily do that by selecting an area of this pull request and clicking the `+-` button (left of the **B**old button).

## New way of releases
I always have been taking all pictures, screenrecords and writing the texts on my own. However I feel that also you who made  the things possible also should have a chance to describe it and share it with the users.  
So if you want to, I invite you to share your words and sections at the release post and/or changelog - if you want.

I would also to make such "proper" release more often (as you can see, I do them ~2 times per year currently). This "releasing the update together" is a first test on how the future of updates looks like.

Again thank you very much for contributing to the project!

----

## Example v2.8 article 
 <https://gsantner.net/blog/2021/07/29/markor-v2.8-todotxt-dialog-contexts-projects-multi-selection.html>

Btw, it's not just the content - also needs title - so we can also decide on the ~2-3 major changes that happened which make up for the update release's headline

grafik

----

## How to make suggestions

Screenshot_20220429-222733  
Click & hold plus button and move mouse down
Screenshot_20220429-222747

### CODE REVIEW

**Comment by gsantner** [comment]
## Git changes

0142c2e1 Renumber list performance improvement, by @harshad1 (PR #1688)
* 7616818b Fix: Template is not applied when creating new file, by @tifish (PR #1669)
* 2102dc6e Enable filename text input filter at rename dialog, by @gsantner closes #1668
* 79bcb47a Cleaner widget logic, update widget on path change, by @harshad1 (PR #1660)
* 7146fb62 Take picture: Change filename template to usual camera format, by @gsantner, #1655
* 0db38d5c Improve showing selection and related functions, by @harshad1 (PR #1658)
* efb9f6a1 Improvements to show selection after search / heading etc, by @harshad1, closes #1325 (PR #1653)
* 40f39e99 Update translations (PR #1654)
* 3514491d Replace discussion reference from Matrix to GitHub discussions, by @gsantner
* 7a3ffd04 Filebrowser: Fix copy/move getCurrentFolder() NullPointer, by @gsantner
* e746478c Fix ArrayIndexOutOfBounds at ShortcutUtils, by @gsantner
* 47de5795 Filemanager: disable search button at root directory, closes #1652, by @gsantner
* 6ac968c5 Base functionality to handle Toolbar (title) clicks & long-clicks, by @gsantner
* 016f2289 Update translations (PR #1649)
* f2aa4b59 Simplify TextAction handling code, by @harshad1 (PR #1626)
* 302b022f markdown: Fix rendered list indent with depth 3, closes #1643, by @fgtham (PR #1651)
* 6014327f Don't run auto-format immediately on document load, by @gsantner
* dedb044f Update translations (PR #1633)
* 285551f2 Fix version of AppIntro dependency, by @gsantner, closes #1632
* 3f7243b0 Feature: Snippets, by @harshad1, closes #437 (PR #1624)
* 6c411b7b Update translations (PR #1627)
* cd3d0fec Automatically set HTML lang attribute, by @fgtham (PR #1623)
* 2c8c682b Update translations (PR #1619)
* 86a55620 TextActions improvements for spacing and save, by @harshad1 (PR #1622)
* c7fdadde textactions: allow setting lower spacing, by @harshad1 (PR #1621)
* b46291f8 wording improvements
* 98e9e83e Markdown: Fix wrong indentation of code blocks in checklists, by @fgtham, closes #1600 (PR #1620)
* 8fb99feb markdown: YAML front-matter improvements, by @gsantner (PR #1597)
* 89d8fec0 Update translations (PR #1616)
* d69e1abf rename string sort 'Sort by' -> sort_by, by @gsantner
* dac68bf4 markdown: Support and display YAML front-matter block contents, by @fgtham @gsantner (PR #1597)
* d9c78be2 Reformat code, optimize imports
* 0b48c2a5 todo.txt: Insert new task start date generally if Auto-Format is enabled, remove special settings option, by @gsantner, closes #1592
* 1a1bdb5b Per-file toggle for auto-format, by @harshad1, closes #1592 (PR #1611)
* e3b1005d Update translations (PR #1608)
* a162bd8b markdown: Add admonition extension, by @fgtham @gsantner (PR #1584)
* 5ac79f9f Markdown: Improve/Fix KaTeX math expression integration, closes #1389, closes #1393, by @fgtham @gsantner (PR #1576)
* 757efd40 ISSUE_TEMPLATE: improve template
* 86742ebc Markdown toc: Frontmatter may be followed by optional empty line, by @fgtham, closes #1246 (PR #1589)
* c539ae7d ISSUE_TEMPLATE: Add dropdown selection for format
* 74c7bb83 Update ISSUE_TEMPLATE
* 0c307d44 Update translations (PR #1571)
* f404ed14 Remove EXTRA_PATH_IS_FOLDER intent parameter everywhere, by @harshad1 (PR #1582)
* 53f87404 Improve document handling, by @harshad1 (PR #1579)
* 0812c174 markdown: Support flowcharts with mermaid, by @fgtham, closes #720 (PR #1581)
* 7b84c407 markdown math/katex: support mhchem (chemistry), by @fgtham (PR #1575)
* 18fa6c64 Fix multiple windows not working in v2.8.5, by @harshad1, closes #1570 (PR #1572)
* 79317e33 (tag: v2.8.6, harshad1/master) Markor v2.8.6 update

**Comment by gsantner** [comment]
## Git changes since 2.8.0

* ddb41a6e Add Markor blog posts markdown documents, by @gsantner
 * 0142c2e1 Renumber list performance improvement, by @harshad1 (PR #1688)
 * 7616818b Fix: Template is not applied when creating new file, by @tifish (PR #1669)
 * 2102dc6e Enable filename text input filter at rename dialog, by @gsantner closes #1668
 * 79bcb47a Cleaner widget logic, update widget on path change, by @harshad1 (PR #1660)
 * 7146fb62 Take picture: Change filename template to usual camera format, by @gsantner, #1655
 * 0db38d5c Improve showing selection and related functions, by @harshad1 (PR #1658)
 * efb9f6a1 Improvements to show selection after search / heading etc, by @harshad1, closes #1325 (PR #1653)
 * 40f39e99 Update translations (PR #1654)
 * 3514491d Replace discussion reference from Matrix to GitHub discussions, by @gsantner
 * 7a3ffd04 Filebrowser: Fix copy/move getCurrentFolder() NullPointer, by @gsantner
 * e746478c Fix ArrayIndexOutOfBounds at ShortcutUtils, by @gsantner
 * 47de5795 Filemanager: disable search button at root directory, closes #1652, by @gsantner
 * 6ac968c5 Base functionality to handle Toolbar (title) clicks & long-clicks, by @gsantner
 * 016f2289 Update translations (PR #1649)
 * f2aa4b59 Simplify TextAction handling code, by @harshad1 (PR #1626)
 * 302b022f markdown: Fix rendered list indent with depth 3, closes #1643, by @fgtham (PR #1651)
 * 6014327f Don't run auto-format immediately on document load, by @gsantner
 * dedb044f Update translations (PR #1633)
 * 285551f2 Fix version of AppIntro dependency, by @gsantner, closes #1632
 * 3f7243b0 Feature: Snippets, by @harshad1, closes #437 (PR #1624)
 * 6c411b7b Update translations (PR #1627)
 * cd3d0fec Automatically set HTML lang attribute, by @fgtham (PR #1623)
 * 2c8c682b Update translations (PR #1619)
 * 86a55620 TextActions improvements for spacing and save, by @harshad1 (PR #1622)
 * c7fdadde textactions: allow setting lower spacing, by @harshad1 (PR #1621)
 * b46291f8 wording improvements
 * 98e9e83e Markdown: Fix wrong indentation of code blocks in checklists, by @fgtham, closes #1600 (PR #1620)
 * 8fb99feb markdown: YAML front-matter improvements, by @gsantner (PR #1597)
 * 89d8fec0 Update translations (PR #1616)
 * d69e1abf rename string sort 'Sort by' -> sort_by, by @gsantner
 * dac68bf4 markdown: Support and display YAML front-matter block contents, by @fgtham @gsantner (PR #1597)
 * d9c78be2 Reformat code, optimize imports
 * 0b48c2a5 todo.txt: Insert new task start date generally if Auto-Format is enabled, remove special settings option, by @gsantner, closes #1592
 * 1a1bdb5b Per-file toggle for auto-format, by @harshad1, closes #1592 (PR #1611)
 * e3b1005d Update translations (PR #1608)
 * a162bd8b markdown: Add admonition extension, by @fgtham @gsantner (PR #1584)
 * 5ac79f9f Markdown: Improve/Fix KaTeX math expression integration, closes #1389, closes #1393, by @fgtham @gsantner (PR #1576)
 * 757efd40 ISSUE_TEMPLATE: improve template
 * 86742ebc Markdown toc: Frontmatter may be followed by optional empty line, by @fgtham, closes #1246 (PR #1589)
 * c539ae7d ISSUE_TEMPLATE: Add dropdown selection for format
 * 74c7bb83 Update ISSUE_TEMPLATE
 * 0c307d44 Update translations (PR #1571)
 * f404ed14 Remove EXTRA_PATH_IS_FOLDER intent parameter everywhere, by @harshad1 (PR #1582)
 * 53f87404 Improve document handling, by @harshad1 (PR #1579)
 * 0812c174 markdown: Support flowcharts with mermaid, by @fgtham, closes #720 (PR #1581)
 * 7b84c407 markdown math/katex: support mhchem (chemistry), by @fgtham (PR #1575)
 * 18fa6c64 Fix multiple windows not working in v2.8.5, by @harshad1, closes #1570 (PR #1572)
 * 79317e33 Markor v2.8.6 update
 * 01ccc836 more: Add buildinfo field for last git commit message, by @gsantner
 * ff7b1356 readme: Improve structure and add more information
 * 371cb8cd readme: Remove codacy github badge
 * ba446f48 preview: Open links to folders in filebrowser, closes #967, by @gsantner
 * 2b5d05c3 ShareInto: Strip common tracking parameters, by @gsantner, closes #1490
 * d4d8f410 Editor: Fix crash at activity state restoring, closes #1565, by @gsantner
 * 440189b0 ShareInto: Fix newline append issue, closes #1569, by @gsantner
 * e1910a1b Update translations (#1560)
 * 3573e63d Replace Google Play badge
 * cd25950c Markor v2.8.5 update release
 * 046d0d31 Zim: Fix Android SDK API level error, by @gsantner
 * c87752a0 Theming: Startup & Activity animations: Quicker, less flashing, closes #1517 #568, by @gsantner
 * 02cd0c36 Filebrowser: Allow shown filename multiline expansion, by @gsantner, discussions #1557 #1558 #1559
 * 0dca87e5 Update translations (PR #1550)
 * 5a5ce82e Reformat XML files, by @gsantner
 * 9cbe72b0 Support day/night theme system, by @harshad1 @gsantner (PR #1543)
 * 9fdc5a1f Update year 2021->2022
 * f7c81393 Debloat: Drop todo.txt huuid experiment, by @gsantner
 * 1dd1cc63 Debloat: Drop redundant launcher shortcut option from editor (new->filebrowser), by @gsantner
 * 2e0b788a Reformat code, by @gsantner
 * 0c27e81c Add launcher shortcuts to folders, by @harshad1 (PR #1549)
 * c43bd37f Set MIN_OVERWRITE_LENGTH=2
 * 456100c2 filebrowser: Change notebook folder menu icon to home, by @harshad1 (PR #1535)
 * 06d114ac Update translations (PR #1546)
 * e02cd233 Bugfix: TextActions document can be null, by @harshad1 (PR #1544)
 * acb073aa Document menu: Swap search<->save button position, by @gsantner
 * c6a4fbe4 Directly show view mode in mainactivity (ToDo, QuickNote), by @gsantner
 * 190410b6 Markdown: soft wrap lines with link, by @gsantner, closes #1536
 * 37cda64d Update translations (PR #1534)
 * ca6f4908 File change state indication & general improvements, by @harshad1 @gsantner (PR #1516)
 * 6b3f1ecc Update translations (PR #1527)
 * c6af143d Privacy: Add option to disallow screenshots, closes #1174  (PR #1514 by @Zoo-M0 @gsantner)
 * 30ca7997 ShareInto: Only trim line breaks when appending, by @gsantner, closes #1526
 * 840d9f5f Update translations (PR #1525)
 * dc9453c0 Combined storage permission check, restart mainactivity & reload all files after grant, by @gsantner (premerge #1359 #1521)
 * 5b2e12f7 Add view mode debugging hint. by @gsantner
 * 4b228db2 Update gradle-wrapper, export activites, by @gsantner (premerge #1359 #1521)
 * f03ac72a Markor update v2.8.4
 * 9a8673af Rename markor baseactivity and use everywhere, move OpenEditor activity to subdir, by @gsantner
 * 5d1ce8a6 Reformat code, by @gsantner
 * 4195eeff Markdown: Support spaces in [li](n ks) and ![im]](ag es), closes #1365 (PR #1510), by @tifish @gsantner
 * 168725e8 Update translations (PR #1515)
 * 06012927 ShareInto: Fix todo.txt entry won't appear, by @gsantner, PR #1495
 * 19a06a47 Add option to control Chrome Custom Tab behaviour, by @gsantner, closes #686 #1385
 * 31d4dbeb todo.txt: Add option to save search queries, by @harshad1 (PR #1467)
 * dba6d23f Update translations (PR #1505)
 * 5f934962 Remove NO_SUGGESTIONS hint from a few dialogs, by @gsantner, closes #1507 #1508
 * 382441c7 Delay inputfilter addition at document load
 * 2f550012 Private AppStorage: Allow to access to files directory only (don't allow access to internals like shared_preferences & databases)
 * a25e9522 Share Into - Format
 * 7f3c4bfd MainActivity todo/quicknote scroll to bottom via setUserVisibleHint
 * d3e91a8a Remove one setDocumentViewVisibility call, #1502
 * 687ef527 ShareInto format/prefix: Improve wording, restore previous defaultvalue
 * 23f79cf0 Share into format/prefix customization, by @harshad1, closes #1368 (PR #1495)
 * 93129cbe Optimize imports & autoformat
 * 152a91c8 Update translations (PR #1491)
 * 06ad871d Improve file I/O, by @harshad1 (PR #1489)
 * 7dc3b05c Move bundled fonts to thirdparty directory, drop 3 fonts to reduce apk size
 * 2492adc6 CI/CD: Makefile: Configurable flavor
 * 38e8c04a CI/CD: Makefile: Add app run job
 * adce9627 Markor v2.8.3
 * ac08c4f7 Add spellchecking capabilities for strings.xml to Makefile & CI/CD, by @gsantner
 * d02b9a0d filemanager: Always load latest fav/recent/popular files when browsing to it,
 * 3d9540e7 ShareInto:: reset modTime after writing, so it's correctly reloaded at time of opening at editor, by @gsantner
 * 7541863d filebrowser::rename file::fix/workaround for Android's filesystem case-insensitive filenaming, by @gsantner, closes #1481
 * 285d0367 Debug builds (aaTest): Properly show version of Markor at 'More' page
 * 95d45315 Markor v2.8.2
 * 1188bf9c Update translations (PR #1480)
 * 07c584e0 Improve file open/close speed, fix file read/write issues, closes #1470, by @harshad1 @gsantner (PR #1473)
 * 4c53b994 Update translations (PR #1477)
 * 520dee63 Zim: follow links to other wiki pages, closes #1223, by @fredericjacob (PR #1437)
 * d92933ed Update translations (PR #1468)
 * a2c4d1f4 Release update v2.8.1, by @gsantner
 * 39220039 Workaround for buggy keyboard at file/folder creation, fixes #1461 (PR #1466 by @gsantner)
 * d68254c6 Update translations (PR #1456)
 * b0d95b6a Remove unused class AppCast
 * bbb0c588 Fix preference-key based buttons in turkish locale, fixes #1425 # 1443 (PR #1455 @gsantner)
 * c9ffc349 Fix recently added languages not displayed accordingly (#1454 by @gsantner)
 * 7ff1f138 Fix language preference: Properly show all languages (PR #1453 by @gsantner)
 * b5513b0d Add language Odia/Oriya (India), Update translations (PR #1446 #1451)
 * 9c4de481 Update translations (PR #1439)
 * f9e1c91e Update translations (PR #1435)
 * a37532bd Update translations (#1431)
 * 3bc417c3 ZimWiki: Syntax highlighting for code, global setting (PR #1434 by @fredericjacob)
 * 2005fa48 Markdown: Individual table of contents level options (PR #1427 by @fredericjacob @gsantner)
 * 129f07e0 Update crowdin config
 * 62f21259 Translation update (PR #1429)
 * 83a4e2f7 filebrowser: Improve performance a lot (speed of file list appearing)
 * aa5b5439 filebrowser:sort: use compareToIgnoreCase for name comparision
 * 7e680df2 Browse todo.txt file by various filters (PR #1408 by @harshad1)
 * a4719b7a todotxt: Save priority on done by pri:X kv (PR #1420 by @harshad1)
 * c500d733 Update translations (PR #1416)
 * 3d5fc9c2 Allow to install APK files from filebrowser on click, by @gsantner (PR #1417)
 * 9e14e176 Update translations (#1405 by @gsantner)
 * f101beda Automatically find FileProvider, by @gsantner
 * fc9269b6 ShareUtils:: mark fileProviderAuthority static
 * 3558f60a Release Markor update v2.8.0 (vc124), by @gsantner

**Comment by gsantner** [comment]
Hello everybody, I want to release a new major Markor update this weekend - this time also with Changelog, blog post etc. This pull request is used as preparation for that release.

I recently always tried to mention you when contributed to the project - so I could exactly reach this point - where I can just copy the git history and paste it somewhere on GitHub and you are automatically there for your contribution. So thanks for helping!

**Now I also invite you to show what you did, worked on, fixed, added as new feature. If you want of course.**

* I added the last Markor release blog post (written in Markdown as well!) to the repository as reference.  --> <https://github.com/gsantner/markor/tree/release/v2.9.0/doc/news>  Also copied the 2.8 release write-up to the new 2.9 write-up. 
* Also added a section to the CHANGELOG - this is short and text only (no image) - summary for app store changelog like f-droid.

To suggest something to be added to the post or changelog, you can easily do that by selecting an area of this pull request and clicking the `+-` button (left of the **B**old button).

## TLDR
I always have been taking all pictures, screenrecords and writing the texts on my own. However I feel that also you who realized the things also should have a chance to say something what you did.  
So if you want to, I invite you to write some thing at the release post,, and/or changelog - if you want.

Again thank you very much for contributing to the project!


----

Screenshot_20220429-222733
Screenshot_20220429-222747

**Comment by gsantner** [comment]
Btw, it's not just the content - also needs title - so we can also decide on the ~2-3 major changes that happened which make up for the update release's headline

**Comment by gsantner** [comment]
@Zoo-M0 @tifish @fredericjacob @fgtham @harshad1  
Thanks for your time on Markor. I want to release an update soon, please check out this pull request and the description above. If you want to share your own words with users, happy to add them

**Comment by gsantner** [comment]
OK I see, there isn't really input/interest by you guys. So I will do it then as usual and pick the things on my own and write some text.

**Comment by harshad1** [comment]
Sorry, busy weekend. Looking now.

File: doc/news/2022-05-01-markor-v2.9.md [hunk 1]
```diff
@@ -41,12 +41,159 @@ This search now also checks contents of files encrypted by Markor.
...
+
+* [Add Markor blog posts markdown documents, by @gsantner](https://github.com/gsantner/markor/commit/ddb41a6e)
+* [Renumber list performance improvement, by @harshad1 (PR #1688)](https://github.com/gsantner/markor/commit/0142c2e1)
+* [Fix: Template is not applied when creating new file, by @tifish (PR #1669)](https://github.com/gsantner/markor/commit/7616818b)
+* [Enable filename text input filter at rename dialog, by @gsantner closes #1668](https://github.com/gsantner/markor/commit/2102dc6e)
+* [Cleaner widget logic, update widget on path change, by @harshad1 (PR #1660)](https://github.com/gsantner/markor/commit/79bcb47a)
+* [Take picture: Change filename template to usual camera format, by @gsantner, #1655](https://github.com/gsantner/markor/commit/7146fb62)
+* [Improve showing selection and related functions, by @harshad1 (PR #1658)](https://github.com/gsantner/markor/commit/0db38d5c)
+* [Improvements to show selection after search / heading etc, by @harshad1, closes #1325 (PR #1653)](https://github.com/gsantner/markor/commit/efb9f6a1)
```
Comment (harshad1) [inline doc/news/2022-05-01-markor-v2.9.md:53]: This one and the one above can be combined.

File: doc/news/2022-05-01-markor-v2.9.md [hunk 2]
```diff
@@ -41,12 +41,159 @@ This search now also checks contents of files encrypted by Markor.
...
+* [Update translations (PR #1654)](https://github.com/gsantner/markor/commit/40f39e99)
+* [Replace discussion reference from Matrix to GitHub discussions, by @gsantner](https://github.com/gsantner/markor/commit/3514491d)
+* [Filebrowser: Fix copy/move getCurrentFolder() NullPointer, by @gsantner](https://github.com/gsantner/markor/commit/7a3ffd04)
+* [Fix ArrayIndexOutOfBounds at ShortcutUtils, by @gsantner](https://github.com/gsantner/markor/commit/e746478c)
+* [Filemanager: disable search button at root directory, closes #1652, by @gsantner](https://github.com/gsantner/markor/commit/47de5795)
+* [Base functionality to handle Toolbar (title) clicks & long-clicks, by @gsantner](https://github.com/gsantner/markor/commit/6ac968c5)
+* [Update translations (PR #1649)](https://github.com/gsantner/markor/commit/016f2289)
+* [Simplify TextAction handling code, by @harshad1 (PR #1626)](https://github.com/gsantner/markor/commit/f2aa4b59)
```
Comment (harshad1) [inline doc/news/2022-05-01-markor-v2.9.md:61]: This is a purely internal change. IMO we don't need to have internal changes (except for performance improvements) in the blogpost. (IMO) we should remove this

File: doc/news/2022-05-01-markor-v2.9.md [hunk 3]
```diff
@@ -41,12 +41,159 @@ This search now also checks contents of files encrypted by Markor.
...
+* [markdown: Fix rendered list indent with depth 3, closes #1643, by @fgtham (PR #1651)](https://github.com/gsantner/markor/commit/302b022f)
+* [Don't run auto-format immediately on document load, by @gsantner](https://github.com/gsantner/markor/commit/6014327f)
+* [Update translations (PR #1633)](https://github.com/gsantner/markor/commit/dedb044f)
+* [Fix version of AppIntro dependency, by @gsantner, closes #1632](https://github.com/gsantner/markor/commit/285551f2)
+* [Feature: Snippets, by @harshad1, closes #437 (PR #1624)](https://github.com/gsantner/markor/commit/3f7243b0)
+* [Update translations (PR #1627)](https://github.com/gsantner/markor/commit/6c411b7b)
+* [Automatically set HTML lang attribute, by @fgtham (PR #1623)](https://github.com/gsantner/markor/commit/cd3d0fec)
+* [Update translations (PR #1619)](https://github.com/gsantner/markor/commit/2c8c682b)
+* [TextActions improvements for spacing and save, by @harshad1 (PR #1622)](https://github.com/gsantner/markor/commit/86a55620)
+* [textactions: allow setting lower spacing, by @harshad1 (PR #1621)](https://github.com/gsantner/markor/commit/c7fdadde)
```
Comment (harshad1) [inline doc/news/2022-05-01-markor-v2.9.md:71]: IMO, this is too minor to include

File: doc/news/2022-05-01-markor-v2.9.md [hunk 4]
```diff
@@ -41,12 +41,159 @@ This search now also checks contents of files encrypted by Markor.
...
+* [Simplify TextAction handling code, by @harshad1 (PR #1626)](https://github.com/gsantner/markor/commit/f2aa4b59)
+* [markdown: Fix rendered list indent with depth 3, closes #1643, by @fgtham (PR #1651)](https://github.com/gsantner/markor/commit/302b022f)
+* [Don't run auto-format immediately on document load, by @gsantner](https://github.com/gsantner/markor/commit/6014327f)
+* [Update translations (PR #1633)](https://github.com/gsantner/markor/commit/dedb044f)
+* [Fix version of AppIntro dependency, by @gsantner, closes #1632](https://github.com/gsantner/markor/commit/285551f2)
+* [Feature: Snippets, by @harshad1, closes #437 (PR #1624)](https://github.com/gsantner/markor/commit/3f7243b0)
+* [Update translations (PR #1627)](https://github.com/gsantner/markor/commit/6c411b7b)
+* [Automatically set HTML lang attribute, by @fgtham (PR #1623)](https://github.com/gsantner/markor/commit/cd3d0fec)
+* [Update translations (PR #1619)](https://github.com/gsantner/markor/commit/2c8c682b)
+* [TextActions improvements for spacing and save, by @harshad1 (PR #1622)](https://github.com/gsantner/markor/commit/86a55620)
```
Comment (harshad1) [inline doc/news/2022-05-01-markor-v2.9.md:70]: Internal change

File: doc/news/2022-05-01-markor-v2.9.md [hunk 5]
```diff
@@ -41,12 +41,159 @@ This search now also checks contents of files encrypted by Markor.
...
+* [Per-file toggle for auto-format, by @harshad1, closes #1592 (PR #1611)](https://github.com/gsantner/markor/commit/1a1bdb5b)
+* [Update translations (PR #1608)](https://github.com/gsantner/markor/commit/e3b1005d)
+* [markdown: Add admonition extension, by @fgtham @gsantner (PR #1584)](https://github.com/gsantner/markor/commit/a162bd8b)
+* [Markdown: Improve/Fix KaTeX math expression integration, closes #1389, closes #1393, by @fgtham @gsantner (PR #1576)](https://github.com/gsantner/markor/commit/5ac79f9f)
+* [ISSUE_TEMPLATE: improve template](https://github.com/gsantner/markor/commit/757efd40)
+* [Markdown toc: Frontmatter may be followed by optional empty line, by @fgtham, closes #1246 (PR #1589)](https://github.com/gsantner/markor/commit/86742ebc)
+* [ISSUE_TEMPLATE: Add dropdown selection for format](https://github.com/gsantner/markor/commit/c539ae7d)
+* [Update ISSUE_TEMPLATE](https://github.com/gsantner/markor/commit/74c7bb83)
+* [Update translations (PR #1571)](https://github.com/gsantner/markor/commit/0c307d44)
+* [Remove EXTRA_PATH_IS_FOLDER intent parameter everywhere, by @harshad1 (PR #1582)](https://github.com/gsantner/markor/commit/f404ed14)
```
Comment (harshad1) [inline doc/news/2022-05-01-markor-v2.9.md:89]: Purely internal change. Shouldn't be included

File: doc/news/2022-05-01-markor-v2.9.md [hunk 6]
```diff
@@ -41,12 +41,159 @@ This search now also checks contents of files encrypted by Markor.
...
+* [Update translations (PR #1619)](https://github.com/gsantner/markor/commit/2c8c682b)
+* [TextActions improvements for spacing and save, by @harshad1 (PR #1622)](https://github.com/gsantner/markor/commit/86a55620)
+* [textactions: allow setting lower spacing, by @harshad1 (PR #1621)](https://github.com/gsantner/markor/commit/c7fdadde)
+* [wording improvements](https://github.com/gsantner/markor/commit/b46291f8)
+* [Markdown: Fix wrong indentation of code blocks in checklists, by @fgtham, closes #1600 (PR #1620)](https://github.com/gsantner/markor/commit/98e9e83e)
+* [markdown: YAML front-matter improvements, by @gsantner (PR #1597)](https://github.com/gsantner/markor/commit/8fb99feb)
+* [Update translations (PR #1616)](https://github.com/gsantner/markor/commit/89d8fec0)
+* [rename string sort 'Sort by' -> sort_by, by @gsantner](https://github.com/gsantner/markor/commit/d69e1abf)
+* [markdown: Support and display YAML front-matter block contents, by @fgtham @gsantner (PR #1597)](https://github.com/gsantner/markor/commit/dac68bf4)
+* [Reformat code, optimize imports](https://github.com/gsantner/markor/commit/d9c78be2)
```
Comment (harshad1) [inline doc/news/2022-05-01-markor-v2.9.md:78]: Minor and internal

File: doc/news/2022-05-01-markor-v2.9.md [hunk 7]
```diff
@@ -41,12 +41,159 @@ This search now also checks contents of files encrypted by Markor.
...
+* [Feature: Snippets, by @harshad1, closes #437 (PR #1624)](https://github.com/gsantner/markor/commit/3f7243b0)
+* [Update translations (PR #1627)](https://github.com/gsantner/markor/commit/6c411b7b)
+* [Automatically set HTML lang attribute, by @fgtham (PR #1623)](https://github.com/gsantner/markor/commit/cd3d0fec)
+* [Update translations (PR #1619)](https://github.com/gsantner/markor/commit/2c8c682b)
+* [TextActions improvements for spacing and save, by @harshad1 (PR #1622)](https://github.com/gsantner/markor/commit/86a55620)
+* [textactions: allow setting lower spacing, by @harshad1 (PR #1621)](https://github.com/gsantner/markor/commit/c7fdadde)
+* [wording improvements](https://github.com/gsantner/markor/commit/b46291f8)
+* [Markdown: Fix wrong indentation of code blocks in checklists, by @fgtham, closes #1600 (PR #1620)](https://github.com/gsantner/markor/commit/98e9e83e)
+* [markdown: YAML front-matter improvements, by @gsantner (PR #1597)](https://github.com/gsantner/markor/commit/8fb99feb)
+* [Update translations (PR #1616)](https://github.com/gsantner/markor/commit/89d8fec0)
```
Comment (harshad1) [inline doc/news/2022-05-01-markor-v2.9.md:75]: Maybe all the Update translations should be combined into one?

File: doc/news/2022-05-01-markor-v2.9.md (hunk 6 above)
Comment (gsantner) [reply doc/news/2022-05-01-markor-v2.9.md:78]: You can ignore all those lines. It is pure git history :smile: .

What matters is the content of the CHANGELOG.md file (there didn't add anything yet). And of course, if you want to say something about some new feature - describing it nicely with optional graphics - example -> <https://gsantner.net/blog/2021/07/29/markor-v2.8-todotxt-dialog-contexts-projects-multi-selection.html>

**Comment by gsantner** [comment]
> Sorry, busy weekend. Looking now.

No stress, also it's if you or somebody else WANTS to do something here.   
I just want to give everybody the chance to also express on their own words what they did :smile: .

**Comment by gsantner** [comment]
note: it is about writing new text, not about the auto-generated git commit changelog.

File: doc/news/2022-05-01-markor-v2.9.md (hunk 7 above)
Comment (gsantner) [reply doc/news/2022-05-01-markor-v2.9.md]: removed translation commits generally

**Comment by gsantner** [comment]
Released

 * project post is here: <https://gsantner.net/blog/2022/05/15/markor-v2.9-snippets-zimwiki-math-daynight-privacy-markdown-todotxt.html>
 * discussion <https://github.com/gsantner/markor/discussions/1701>
 * Release/Download <https://github.com/gsantner/markor/releases/tag/v2.9.0>

**Comment by DarkPark** [comment]
Why the new version is not available in the google market https://play.google.com/store/apps/details?id=net.gsantner.markor ?

**Comment by gsantner** [comment]
I dont update Google Play anymore. There are 2 installation options listed. As of why, read through what you can find in github markor comments. I don't wanna repeat.
